File: valid-anagram/valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        
        int ls = s.length();
        int lt = t.length();
        
        if(ls != lt){
            return false;
        }
        
        char[] cs = s.toCharArray();
        char[] ct = t.toCharArray();
        
        Arrays.sort(cs);
        Arrays.sort(ct);
        
        String newS = new String(cs);
        String newT = new String(ct);
        
        if(newS.equals(newT)){
            return true;
        }
        
        return false;
        
    }
}

File: valid-anagram/valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        
        if(len(s) != len(t)):
            return False
        
        d = dict();
        for s1 in s:
            d[s1] = s.count(s1)
        
        for t1 in t:
            if d.get(t1) == None:
                return False
            elif d.get(t1) != t.count(t1):
                return False
        return True